DMCompiler/DMStandard/_Standard.dm

```diff
@@ -133,29 +133,91 @@ proc/block(var/atom/Start, var/atom/End)
 
 	return atoms
 
-proc/range(Dist, atom/Center = usr)
+// TODO: Investigate "for(var/turf/T in range(Dist, Center))"-style weirdness that BYOND does. It's a center-out spiral and we need to replicate that.
```

Comment:
By "weirdness" here are you referring to the spiral or something more specific to the for loop?

Reply:
Former.